Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -719,6 +719,23 @@ public static class Defaults
                 Statistic = Statistic.Average,
                 Namespace = AwsNamespace.Elb,
                 TreatMissingData = TreatMissingDataConstants.NotBreaching
+            },
+            new AlarmDefinition
+            {
+                Name = "HealthyHostCountLow",
+                Metric = "HealthyHostCount",
+                Period = TimeSpan.FromMinutes(5),
```

Comment:
could do 1 min? 
I see the others are 5 but not sure why

Reply:
oh, it looks like they are every 5 minutes unless you change a config somewhere else: https://docs.aws.amazon.com/autoscaling/ec2/userguide/as-instance-monitoring.html

Reply:
oh, that's autoscaling and this is ELB, ignore me

Reply:
I've still gone for 1 min.  Would rather fail sooner.

Reply:
Looks like ELB is 1 minute resolution, so it should work to change it to 1

https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/elb-cloudwatch-metrics.html

> If there are requests flowing through the load balancer, Elastic Load Balancing measures and sends its metrics in 60-second intervals.